src/main/java/com/example/application/views/PrivateJavaView.java

```diff
@@ -29,6 +32,9 @@ public PrivateJavaView(BankService bankService, SecurityUtils utils) {
     }
 
     private void updateBalanceText() {
+        if (utils.getAuthenticatedUser() == null) {
```

Comment:
How can this happen as there is `@RolesAllowed("user")` on the class?

Reply:
It could happen before I added the security check. It shouldn't happen anymore, so I'll remove it.